src/fileActions.jsx

```diff
@@ -139,11 +124,11 @@ export const ConfirmDeletionDialog = ({
         const selectedItem = selected.length === 1
             ? selected[0]
             : currentDirectory;
-        if (selectedItem.type === "file") {
+        if (selectedItem.type === "reg") {
             modalTitle = cockpit.format(_("Delete file $0?"), selectedItem.name);
-        } else if (selectedItem.type === "link") {
+        } else if (selectedItem.type === "lnk") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240126-113835-57de325b-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)